Fixes #69, #62: Added Techwin SNC-4241BE IP Camera support and added SNCN2900 User Manual

jhallard opened the pull request.

jhallard committed:
Fixes #69: Added Techwin SNC-4241BE IP Camera support

jcmaslan commented on cameras/paths.csv:
```diff
@@ -19,6 +19,7 @@ Digital Watchdog,nvr,VMAX IP Plus,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{pass
 Amcrest,nvr,NV1104,(90:02:A9),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,0;1;2;3,,,,,admin,admin,,http://amcrest.com/fileuploader/download/download/?d=0&file=custom%2Fupload%2FAmcrest_NVR_User_Manual_V3.3.pdf
 Amcrest,camera,IP2M-841,(90:02:A9|9C:8E:CD),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel=1&subtype={{stream}},H.264,554,0;1,,,,1,0,admin,admin,,https://support.amcrest.com/hc/en-us/article_attachments/203939917/Amcrest_PT_Camera_User_Manual_v1.1.7.pdf
 Samsung Techwin,camera,SNH-E6411BN;SNH-V6414BN;SNH-E6440BN;SNH-V6410BN,(00:09:18|00:16:1C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/profile{{stream}}/media.smp,H.264,554,3;4;5,,Low 360p;Mid 720p;High 1080p,,3,5,admin,4321,,https://www.samsungsv.com/Download/SNH-P6410BN-User-Manual.pdf
+Samsung Techwin,camera,SNC-4241BE,(00:16:1C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/ch01/{{stream}},H.264,554,1;0;2,,Stream {{stream}},,2,0,admin,admin,,https://www.samsungsv.com/Download/SNK-D5081_User_Manaul.pdf
```
now that we have the set_stream_0 config command, please make the streams `0;1;2` so that the default is stream `0`. The names of those three streams are `Main Stream`, `Sub Stream` and `Mobile Stream` respectively.

jhallard replied:
done with e72221d

jhallard committed:
Fixes #62: Added user manual for SCNC2900/SCNC3905

Also, added SCNC2900 as a possible name for cameras with this OUI/path

jhallard committed:
Merge branch 'john' of https://github.com/CamioCam/rtsp into john

jhallard committed:
Fixes #73: Add Vivotek CC8130 camera information

This won't work until the cams repo is changed to support
no-default-username (an open PR exists for this)

jhallard committed:
Changed stream-default and names for Techwin SNC-4241 camera

jcmaslan commented on cameras/paths.csv:
```diff
@@ -32,3 +33,4 @@ Arecont,camera,AV Models,(00:1A:07),rtsp://{{username}}:{{password}}@{{ip_addres
 Microseven,camera,M7XXX,(10:83:D2),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/1{{stream}},H.264,554,1;2;3,,High;Medium;Low,,3,1,admin,password,,http://www.microseven.com/manuals/M7xxx-NetworkCamera-UserGuide_2016_FINAL.pdf
 Grandstream,nvr,GXV3504,(00:0B:82),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{channel}},H.264,554,,0;1;2;3;4;5;6;7,,Channel {{channel}},,,admin,admin,,http://www.voipsupply.com/downloads/dl/file/id/8911/grandstream_gxv3504_manual.pdf
 Grandstream,camera,GXV3674_[F]HD_VF,(00:0B:82),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}},H.264,554,4,,Stream {{stream}},,4,4,admin,admin,,http://www.grandstream.com/sites/default/files/Resources/GXV3674hd_fhd_v2_usermanual_english.pdf
+Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}}.sdp,H.264,554,1;2;3,,live;live2;liveany;,,live2,live,,,,http://download.vivotek.com/downloadfile/downloads/usersmanuals/cc8130manual_en.pdf
```
The `streams` have to be `live;live2;liveany` as the actual stream values (not just the descriptions).

jhallard replied:
check now? I set the descriptions to be `Stream 1;Stream 2;Stream 3`.

jhallard committed:
1;2;3 --> live;live2;liveany for Vivotek camera

jcmaslan commented:
LGTM

jhallard commented:
Cameras app is updated so no username is required now. Merging and deploying.
